src/v8/v8.cc

```diff
@@ -66,6 +68,14 @@ class V8 : public WasmVm {
             const std::unordered_map<uint32_t, std::string> &function_names) override;
   std::string_view getPrecompiledSectionName() override;
   bool link(std::string_view debug_name) override;
+  void terminateExecution() override {
```

Comment:
Could you split declaration and implementation, like other functions in the code?

Also, the placement should match that in the `wasm_vm.h`.